pkg/api/controller.go

```diff
@@ -1884,36 +1884,54 @@ func (c *Controller) GetStorageConfig(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// TODO (gilo): is StorageID relevant here?
-	storageConfig, err := c.getStorageConfig("")
-	if c.handleAPIError(r.Context(), w, r, err) {
-		return
-	}
-
-	writeResponse(w, r, http.StatusOK, storageConfig)
+	storageCfg, _ := c.getStorageConfig(config.SingleBlockstoreID)
+	writeResponse(w, r, http.StatusOK, storageCfg)
 }
 
-func (c *Controller) getStorageConfig(storageID string) (apigen.StorageConfig, error) {
-	info, err := c.BlockAdapter.GetStorageNamespaceInfo(storageID)
-	if err != nil {
-		return apigen.StorageConfig{}, err
+func (c *Controller) getStorageConfig(storageID string) (*apigen.StorageConfig, error) {
+	storage := c.Config.StorageConfig().GetStorageByID(storageID)
+	if storage == nil {
+		return nil, config.ErrNoStorageConfig
 	}
-	defaultNamespacePrefix := swag.String(info.DefaultNamespacePrefix)
-	if c.Config.GetBaseConfig().Blockstore.DefaultNamespacePrefix != nil {
-		defaultNamespacePrefix = c.Config.GetBaseConfig().Blockstore.DefaultNamespacePrefix
+	info := c.BlockAdapter.GetStorageNamespaceInfo(storageID)
+	if info == nil {
+		c.Logger.Error("no storage namespace info found for id: %s", storageID)
+		return nil, config.ErrNoStorageConfig
+	}
+
+	defaultNamespacePrefix := storage.GetDefaultNamespacePrefix()
+	if defaultNamespacePrefix != nil {
+		info.DefaultNamespacePrefix = *defaultNamespacePrefix
 	}
-	storageConfig := apigen.StorageConfig{
-		BlockstoreType:                   c.Config.GetBaseConfig().Blockstore.Type,
+	return &apigen.StorageConfig{
+		BlockstoreDescription: swag.String(storage.BlockstoreDescription()),
```

Comment:
Should also add here `BlockstoreID` - 
The UI needs it for rendering.

Reply:
Adding it as part of the storageConfigList response